src/EnergyPlus/GeneralRoutines.cc

```diff
@@ -616,6 +631,50 @@ ControlCompOutput(
 
 }
 
+bool
+BBConvergeCheck(
+	int const SimCompNum,
+	Real64 const MaxFlow,
+	Real64 const MinFlow
+)
+{
+
+	// FUNCTION INFORMATION:
+	//       AUTHOR         Rick Strand
+	//       DATE WRITTEN   November 2017
+	
+	// PURPOSE OF THIS SUBROUTINE:
+	// This is an additional check for the radiant/convective baseboard units
+	// to see if they are converged or the flow is sufficiently converged to
+	// procede with the simulation.  With the radiant component to these systems,
+	// the impact on the load met is more difficult to calculate and the impact
+	// on the actual system output is not as well behaved as for convective
+	// systems.  This additional check avoids excessive iterations and max
+	// iteration warnings and provides sufficiently converged results.  It is
+	// only called from ControlCompOutput.
+
+	// Return Value
+	bool BBConvergeCheck;
+	
+	// SUBROUTINE PARAMETER DEFINITIONS:
+	static Real64 const BBIterLimit( 0.00001 );
+
+	if ( SimCompNum != 5 && SimCompNum != 6 ) {
+		// For all zone equipment except radiant/convective baseboard (steam and water) units:
+		BBConvergeCheck = false;
+	} else {
+		// For steam and water radiant/convective baseboard units:
+		if ( ( MaxFlow - MinFlow ) > BBIterLimit ) {
```

Comment:
There's no reason to assume max could ever be below min here, right?  At first I was thinking about putting an absolute value call around that difference, but really, that sort of condition would probably reveal a bigger problem, and we shouldn't put a band-aid over it for no reason.  Nothing to see here then...carry on...

Reply:
Right...if Max is smaller than Min something major is wrong and it will probably not even get here.